libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Rewrite.hs

```diff
@@ -50,7 +50,7 @@ import qualified Data.Map.Strict as Map
 import Data.Set (Set)
 import qualified Data.Set as Set
 import Data.Void (Void)
-import Debug.Trace
+import qualified Debug.Trace as Debug
```

Comment:
Looks like an unused import. This module needs to be cleaned up at some point and disables warnings at the top of the module will need to go to.

Reply:
It was easy to do, so I pushed a couple extra commits for this. I also rebased on `master`.